Blackjack.py

```diff
@@ -0,0 +1,282 @@
+        print(f"\nDealer's hand:\nhidden\n{dealer.all_cards[1]}")
+        print("\nPlayer's hand:", *player.all_cards, player, sep="\n")
+
+
+    def show_all(player, dealer):
+        print("Lay it all on the floor , please yo!")
+        print("\nDealer's Hand: \n", *dealer.all_cards, dealer, sep="\n")
+        print("\nPlayer's Hand: \n", *player.all_cards, player, sep="\n")
+
+
+    def player_busts(player, player_chips):
+        print("*****************************")
+        print("Player busts")
+        print("*****************************")
+        player_chips.lose_bet()
+        # player.all_cards = []
+
+
+    def player_wins(player, player_chips):
+        print("*****************************")
+        print("Player wins!")
+        print("*****************************")
+        player_chips.win_bet()
+        # player.all_cards = []
+
+
+    def dealer_busts(dealer, dealer_chips):
+        print("*****************************")
+        print("Dealer busts")
+        print("*****************************")
+        dealer_chips.lose_bet()
+        # dealer.all_cards = []
+
+
+    def dealer_wins(dealer, dealer_chips):
+        print("*****************************")
+        print("Dealer Wins!")
+        print("*****************************")
+        dealer_chips.win_bet()
+        # dealer.all_cards = []
+
+
+    def push():
+        print("****************************************************")
+        print("Its a tie, both Player and Dealer Push Bay-Bee")
+        print("****************************************************")
+        pass
+
+
+    the_deck = Deck()
+    # player = Hand()
+    # dealer = Hand()
+
+    player_chips = Chips()
+    dealer_chips = Chips()
+
+    round = 1
+
+    while True:  # This is the overall Blackjack game, until all cards in deck are gone or player quits
+
+        player = Hand()
+        dealer = Hand()
+
+        print(
+            f"Round {round}: The deck has {len(the_deck.all_cards)}\nPlayer has {player_chips.total} chips\nDealer has {dealer_chips.total} chips")
+        playing = True
+
+        take_bet(player_chips)
+        dealer_chips.bet = player_chips.bet
+
+        the_deck.shuffle()
+
+        for x in range(0, 2):
+            player.add_card(the_deck.deal())
+
+        for y in range(0, 2):
+            dealer.add_card(the_deck.deal())
+
+        show_some(player, dealer)
+
+        while playing and player.player_value != 21:  # single game until either the player or dealer wins the bet
+
+            # player turn
+
+            hit_or_stand(the_deck, player)
+
+            show_some(player, dealer)
+
+            if player.player_value >= 21:
+                break
+
+                # player bust haha
+        if player.player_value > 21:
+            player_busts(player, player_chips)
+
+        if player.player_value == 21:
+            print("MF Blackjack!!!!!, Dealer Loses")
+            player_wins(player, player_chips)
+            dealer_chips.lose_bet()

```

Comment:
Perhaps this could go into a `collect_player_bets` function.

And then a `collect_dealer_bets` function for that portion.